[chore] Add connecting to Strapi section to CONTRIBUTING (#836)
strapi/design-system: README.md

```sh
$ git reset --hard
```

> ## Connecting to Strapi

Reviewer: Wdyt about 
```suggestion
## Linking the design-system in your application
```

> If you want to analyze the admin bundle and the impact each library has on the over all application you can link this repo to your copy of [@strapi/strapi](https://github.com/strapi/strapi).

Reviewer 2: ```suggestion
If you want to analyze the admin bundle and the impact each library has on the overall application, you can link this repo to your copy of [@strapi/strapi](https://github.com/strapi/strapi).
```

First, run `yarn build` in `strapi-design-system/packages/strapi-design-system` to generate the bundle.

> In your copy of Strapi you can link the design system using either a [relative path](#relative-path) or [yarn link](#yarn-link).

Reviewer 2: ```suggestion
In your copy of Strapi, you can link the design system using either a [relative path](#relative-path) or [yarn link](#yarn-link).
```

### Relative path

Replace the version number in both `strapi/packages/core/admin/package.json` and `strapi/packages/core/helper-plugin/package.json` with the relative path to your copy of the design system:

```
"@strapi/design-system": "link:../../../../strapi-design-system/packages/strapi-design-system"
```

### Yarn link

> Alternatively, you can use [`yarn link`](https://classic.yarnpkg.com/lang/en/docs/cli/link/) by first running `yarn link` in `strapi-design-system/packages/design-system` and then `yarn link @strapi/design-system` in both `strapi/packages/core/admin` and `strapi/packages/core/helper-plugin`. With this approach no changes need to be made to the `package.json`

Reviewer 2: ```suggestion
Alternatively, you can use [`yarn link`](https://classic.yarnpkg.com/lang/en/docs/cli/link/) by first running `yarn link` in `strapi-design-system/packages/design-system` and then `yarn link @strapi/design-system` in both `strapi/packages/core/admin` and `strapi/packages/core/helper-plugin`. With this approach, no changes need to be made to the `package.json`
```

```sh
$ git reset --hard
```

## Linking the design system

### Strapi monorepo

To link the design system to the Strapi monorepo follow the steps outlined in the [contributor documentation](https://contributor.strapi.io/core/admin/link-strapi-design-system)

### React application

First, run `yarn build` in `strapi-design-system/packages/strapi-design-system` to generate the bundle.

You can link the design system using either a [relative path](#relative-path) or [yarn link](#yarn-link).

### Relative path

In the package.json of your React application replace the version number with the relative path to your copy of the design system:

```

> "@strapi/design-system": "link:../../../../strapi-design-system/packages/strapi-design-system"

Reviewer: ```suggestion
"@strapi/design-system": "link: <relative-path>/strapi-design-system/packages/strapi-design-system"
```
